addons/pos_restaurant/static/src/app/floor_screen/floor_screen.js

```diff
@@ -363,7 +363,8 @@ export class FloorScreen extends Component {
     }
     async addFloor() {
         const { confirmed, payload: newName } = await this.popup.add(TextInputPopup, {
-            title: this.env._t("Floor Name"),
+            title: this.env._t("New Floor"),
+            placeholder: "Floor name",
```

Comment:
```suggestion
            placeholder: this.env._t("Floor name"),
```

Always translate terms.